packages/features/bookings/Booker/components/DatePicker.tsx

```diff
@@ -38,6 +39,7 @@ export const DatePicker = ({
     shallow
   );
   const nonEmptyScheduleDays = useNonEmptyScheduleDays(schedule?.data?.slots);
+  const { timezone } = useTimePreferences();
```

Comment:
Hey @vijayraghav-io, can we do the same directly inside the `DatePickerComponent` instead of passing it from here?

Reply:
Yes, i started with having useTimePreferences() inside <Days> component, everything works fine w.r.t this fix and functionality.

But when i ran the complete tests, the test "date-override-list.test.tsx" fails without even showing proper error message.
I found it in really hard way spending so much time, then finally found that - useTimePreferences() inside Days() component or even inside getTodaysDateInTimeZone.ts was causing this test to fail.

components/DatePicker.tsx is the nearest parent component which was not throwing this test fail.